custom/icds_reports/management/commands/queue_bihar_child_delivery_forms.py

```diff
@@ -0,0 +1,45 @@
+from datetime import date
+from django.core.management.base import BaseCommand
+from corehq.apps.userreports.tasks import _get_config_by_id
+from corehq.apps.userreports.util import get_table_name
+from corehq.apps.userreports.models import AsyncIndicator
+from django.db import connections
+
+from dimagi.utils.chunked import chunked
+
+
+class Command(BaseCommand):
+    help = "Rebuild Bihar delivery forms"
+
+    def handle(self, *args, **kwargs):
+        delivery_config = _get_config_by_id('static-icds-cas-static-child_delivery_forms')
+        table_name = get_table_name('icds-cas', 'static-child_delivery_forms')
+        # sort by supervisor_id and doc_id to improve the performance, sorting is needed to resume the queueing
+        # if it fails in between.
```

Comment:
I don't see a mechanism for resuming, how will you do that?

Reply:
Added the resume mechanism.